src/index.tsx

```diff
@@ -48,6 +53,94 @@ function Updaters() {
     )
 }
 
+const Input = styled.input<{ error?: boolean }>`
+    border-radius: 5px;
+    width: 100%;
+`
+const Wrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 30px;
+    padding: 10%;
+    justify-content: center;
+    align-items: center;
+`
+
+function CustomApp() {
+    const [auth, setAuth] = useState(false)
+    const PASSWORD = 'gdbetatest'
+    const PASSWORD_HASH = '$2a$10$V9DPoPvZtRpg9t23wzl5c.jYYyG5VJdJx/pvBJy61WmN/01rkSfSm'
+
+    useEffect(() => {
+        const passMatch = window.location.search.match(/\bpass=\w+,\b/)
+        if (passMatch && passMatch[0]) {
+            let pass = passMatch[0].split('=')[1]
+            pass = pass.substring(0, pass.length - 1)
+
+            if (pass === PASSWORD) setAuth(true)
+        } else if (localStorage.getItem('pass') === PASSWORD_HASH) setAuth(true)
+    }, [])
+
+    const [value, setValue] = useState('')
+
+    const handleChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+        setValue(e.target.value)
+    }
+
+    const handleSubmit = () => {
+        if (value) {
+            if (value === PASSWORD) {
+                localStorage.setItem('pass', PASSWORD_HASH)
+                setAuth(true)
+            } else {
+                alert('Incorrect password')
+            }
+        }
+    }
+
+    return (
+        <>
+            {auth ? (
+                <App />
+            ) : (
+                <Modal isOpen={true} onDismiss={() => null}>
+                    <Wrapper>
+                        <Image src={LogoImg} alt="logo" width="100px" />
+                        <Text>
+                            Enter password to continue to <b>GoodDollar</b>
+                        </Text>
```

Comment:
text with not attributes? does it still has the correct look and fill?

Reply:
Yes it looks fine